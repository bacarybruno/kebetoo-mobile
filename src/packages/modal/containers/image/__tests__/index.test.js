import setupTest from 'Kebetoo/src/config/jest-setup'

import ImageModal from '../index'

jest.mock('@react-navigation/native', () => ({
  ...jest.requireActual('@react-navigation/native'),
  useRoute: () => ({
    params: {
      source: {
        uri: 'jest://fake-image.png',
      },
      width: 100,
      height: 100,
    },
  }),
}))

const givenImageModal = setupTest(ImageModal)({
  navigation: {
    setOptions: jest.fn(),
  },
})

it('renders ImageModal', () => {
  const { wrapper } = givenImageModal()
  expect(wrapper.toJSON()).toMatchSnapshot()
})
