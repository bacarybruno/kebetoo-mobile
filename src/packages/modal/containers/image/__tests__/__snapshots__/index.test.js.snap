// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders ImageModal 1`] = `
<View
  style={
    Object {
      "backgroundColor": "#000",
      "flex": 1,
      "justifyContent": "center",
    }
  }
>
  <Image
    source={
      Object {
        "uri": "http://localhost:1337/fake-image.png",
      }
    }
    style={
      Object {
        "aspectRatio": 1,
      }
    }
  />
</View>
`;
